topik/pemrograman-kompetitif-dasar/03-struktur-data-dasar/pemrograman-kompetitif-dasar_03-struktur-data-dasar.tex

```diff
@@ -11,43 +11,265 @@
 \end{frame}
 
 \begin{frame}
-\frametitle{Panduan Konten}
+\frametitle{Pendahuluan}
+Melalui dokumen ini, kalian akan:
 \begin{itemize}
-  \item Pengertian struktur data \& apa pentingnya
-  \item Review: tunjukkan bahwa array adalah contoh struktur data sederhana yang mendukung operasi read \& update secara random access
-  \item Bahas linked list, tekankan apa keuntungan/kerugian dari linked list dibandingkan dengan array
-  \item Bahas stack dan queue, bisa dimulai dengan diberikan permasalahan, lalu cari struktur data yang tepat kira-kira apa. Contohnya sederhana saja, misalnya tumpukan piring cucian dan antrian kantor pajak
-  \item Jelaskan tentang Binary Search Tree, supaya ada variasi yang lebih jelas. Biasanya kalau hanya belajar stack/queue/linked list, orang tidak terlalu paham abstraksi dari struktur data. Cukup jelaskan operasi insert dan find saja. Operasi delete cukup disebut, dan beri link supaya peserta bisa baca kalau berminat.
+	\item Mengenal beberapa macam struktur data dasar
```

Comment:
ini indentasinya pake tab ya
tolong diganti jadi spasi 2x aja, supaya lebarnya selalu sama di mana2.
biisa di-find and replace supaya jadi spasi 2x aja